### ISSUE TITLE
Worker Info

### ISSUE THREAD

**Issue by ayush25102001** [KaizenGirl1111/GSGrihSangini#33]
Currently the application does not provide facility for worker registration and login.
I will create:
 1->Worker Schema:
       Fields:
                a)Name
                b)Contact details
                c)CategoryOfService (Gardener,Plumber,sweeper,etc)
                d)password (Everyone can set password of their own) 
2->Indivisual login and signUp forms for worker category.
Can I work on this issue as a part of Gssoc'22

**Comment by KaizenGirl1111** [comment]
@ayush25102001 A person had made a dummy database of workers. You can definitely make this one! Just keep us posted with your progress and ask doubt to mentor(reviewer) if any

**Comment by ayush25102001** [comment]
> @ayush25102001 A person had made a dummy database of workers. You can definitely make this one! Just keep us posted with your progress and ask doubt to mentor(reviewer) if any

Mam I will create another issue for the backend and Db part for workers once this is done
